sdk/keyvault/azsecrets/CHANGELOG.md

```diff
@@ -1,12 +1,13 @@
 # Release History
 
-## 0.4.1 (Unreleased)
+## 0.5.0 (Unreleased)
```

Comment:
For preview, we've been trying to follow the pattern that breaking changes bump the minor version.  This doesn't look to be breaking, at least for compile time.  Is this being considered breaking for runtime?

Reply:
In runtime I think so, the request body will change with this code change.

Reply:
I see.  Then should the comment be moved to the `Breaking Changes` category?  It almost makes me think that we could benefit from a `Runtime Changes` category to differentiate between compile-time and runtime breaking changes (maybe we just note this is runtime only in the comment).

Reply:
Agree that it should be under breaking changes. I don't think we'll have a lot of changes that fall under a "Breaking Change but only at runtime" and for now should just note it in the changelog entry.